composer.json

```diff
@@ -160,7 +160,8 @@
     "knplabs/knp-paginator-bundle": "^5.4.2",
     "masterminds/html5" : "^2.7",
     "league/flysystem-bundle" : "^2.0",
-    "spatie/image-optimizer": "^1.6.2"
+    "spatie/image-optimizer": "^1.6.2",
+    "opis/closure": "^3.5"
```

Comment:
Note that this library is **not** compatible with PHP 8.1 (even if the `composer.json` states otherwise) and probably never will be (even if there's an experimental `v4` using FFI)!

You can use [Laravel's fork](https://github.com/laravel/serializable-closure) or the [experimental FFI version](https://github.com/opis/closure/issues/59) of this package (I'm not sure if it's actively developed, though).

See: opis/closure#128, opis/closure#127, opis/closure#125, opis/closure#124, opis/closure#120, opis/closure#114, opis/closure#107, opis/closure#104, ...